app/gateway/2.7.x/configure/auth/kong-manager/oidc.md

```diff
@@ -17,25 +17,26 @@ manually enable the **Plugin**; the configuration alone will enable
 The following is an example using Google as the IdP and serving Kong Manager
 from its default URL, `http://127.0.0.1:8002`.
 
-(The `admin_gui_auth_config` value must be valid JSON.)
+(The `admin_gui_auth_conf` value must be valid JSON.)
 
 ```
 enforce_rbac = on
 admin_gui_auth=openid-connect
 admin_gui_session_conf = { "secret":"set-your-string-here" }
 admin_gui_auth_conf={                                      \
   "issuer": "https://accounts.google.com/",                \
+  "admin_claim": "email",                  \ 
   "client_id": ["<ENTER_YOUR_CLIENT_ID>"],                 \
-  "client_secret": ["<ENTER_YOUR_CLIENT_SECRET_HERE>"],    \
-  "consumer_by": ["username","custom_id"],                 \
+  "client_secret": ["<ENTER_YOUR_CLIENT_SECRET_HERE>"],    \  
+  "admin_by": "username",                                \ 
```

Comment:
the `admin_by` also the new `admin_gui_auth_conf` option, and currently we only support admin_by = username. so I would think either explains what is `admin_by` and it's current limitation, or don't expose it to customer(it's not a requried option).

Reply:
When you say "new `admin_gui_auth_conf` option do you mean the `admin_claim` field? It's in there, at line 28. But we could re-order them, sure. And remove `admin_by` -- I wasn't sure about including that one anyway :D

Reply:
yes, I meant we can remove the `admin_by`, since it's has to be and has a default value as `username` and customer don't need to specify that.